### PR TITLE
Add r-gutenbergr

### PR DESCRIPTION
**Pull request by hliebert**
Adding the r-gutenbergr package. I used the conda R skeleton helper.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hliebert** [comment]
@conda-forge-admin, please ping team

**Comment by hliebert** [comment]
@conda-forge-admin, please ping team

Sorry for being pushy. I am relying on this package for a course I am teaching soon and would be very happy if the request could be merged.

File: recipes/r-gutenbergr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - r-urltools
+
+test:
+  commands:
+    - $R -e "library('gutenbergr')"           # [not win]
+    - "\"%R%\" -e \"library('gutenbergr')\""  # [win]
+
+about:
+  home: https://docs.ropensci.org/gutenbergr/, https://github.com/ropensci/gutenbergr
+  license: GPL-2.0-or-later
```
Comment (cbrueffer) [inline recipes/r-gutenbergr/meta.yaml:51]:
```suggestion
  license: GPL-2.0-only
```

**Review by cbrueffer — COMMENTED** [review]
Please apply the one license suggestion, otherwise it look good. I'll merge this once the little change is in.

**Comment by hliebert** [comment]
@cbrueffer Applied it. Thank you very much!

**Pull request merged**